libpod/filters/containers.go

```diff
@@ -0,0 +1,157 @@
+package lpfilters
+
+import (
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/containers/libpod/libpod"
+	"github.com/containers/libpod/libpod/define"
+	"github.com/containers/libpod/pkg/timetype"
+	"github.com/containers/libpod/pkg/util"
+	"github.com/pkg/errors"
+)
+
+// GenerateContainerFilterFuncs return ContainerFilter functions based of filter.
+func GenerateContainerFilterFuncs(filter, filterValue string, r *libpod.Runtime) (func(container *libpod.Container) bool, error) {
```

Comment:
Why doesn't this return `[]libpod.ContainerFilter`?

Reply:
because the iteration is outside this

Reply:
I'm also really iffy about this being in libpod - seems to be moving CLI parsing into the Libpod API

Reply:
Pod/volume filters seem to be in cmd/podman/shared